test-apps/testpurchasesuiandroidcompatibility/build.gradle.kts

```diff
@@ -48,10 +49,16 @@ android {
     }
 }
 
+emerge {
+    apiToken.set(System.getenv("EMERGE_API_TOKEN"))
+}
+
 dependencies {
     implementation(project(":purchases"))
     implementation(project(":ui:revenuecatui"))
 
+    androidTestImplementation(libs.emerge.snapshots)
```

Comment:
Hmm could we move the plugin/dependencies to the `ui:revenuecatui` module, since that's where our previews live. Or does it need to be part of an application module?

Reply:
Emerge's snapshots need to run on an application module as we rely on an instrumentation test we manage to run the snapshots. 

Given this module depends directly on the `:ui:revenuecatui` module and snapshots uses a debug variant, all code should be present from that module which will give us all Previews automatically.